rasa/core/actions/forms.py

```diff
@@ -64,6 +64,7 @@ def from_entity(
         not_intent: Optional[Union[Text, List[Text]]] = None,
         role: Optional[Text] = None,
         group: Optional[Text] = None,
+        global_not_intent: Optional[Text] = None,
```

Comment:
After looking through the code I don't think this method is ever called with `intent` or `not_intent`. I think they must be legacy. It seems like `intent_is_desired` is the method you actually have to look at. 
Also, another option to passing the `global_not_intent` around, is to just update the `not_intent` list for each  `slot_mapping` in the domain when it's initialized.